packages/office-ui-fabric-react/src/components/Button/ButtonCore/ButtonCore.scss

```diff
@@ -31,18 +31,12 @@
     border-color: $ms-color-contrastWhiteSelected;
   }
 
-  &:disabled,
-  &.disabled {
+  &.is-disabled {
```

Comment:
Can we use attribute selectors eg. `aria-disabled=true` to provide styles wherever possible? It provides multiple benefits:
1. Provides symantic to our CSS - CSS is more readable and meaningful.
2. Less chances to accessibility regression - any regression in accessibility due to future changes can be caught upfront.